code/modules/species/species.dm

```diff
@@ -647,29 +647,28 @@ The slots that you can use are found in items_clothing.dm and are the inventory
 	var/list/holding = list(target.get_active_hand() = 60, target.get_inactive_hand() = 30)
 
 	var/skill_mod = 10 * attacker.get_skill_difference(SKILL_COMBAT, target)
-	var/state_mod = attacker.melee_accuracy_mods() - target.melee_accuracy_mods()
-	var/stim_mod = target.chem_effects[CE_STIMULANT]
-	var/push_mod = min(max(1 + attacker.get_skill_difference(SKILL_COMBAT, target), 1), 3)
-	if(target.a_intent == I_HELP)
-		state_mod -= 30
+	var/state_mod = attacker.melee_accuracy_mods() - target.melee_accuracy_mods() + ((target.a_intent == I_HELP) ? 15 : 0)
```

Comment:
I don't really get it. Why was the intent logic switched? What's going on in this file, in general?